docs/versioned_docs/version-3.2.0/icicle/primitives/poseidon2.md

```diff
@@ -26,66 +27,74 @@ You can also use your own set of constants as shown [here](https://github.com/in
 
 ### Rust API
 
-This is the most basic way to use the Poseidon2 API.
+This is the most basic way to use the Poseidon2 API. See [examples/poseidon2](examples/rust/poseidon2)
```

Comment:
This is a broken link. Since it links outside of the docs folder (the site) it needs a full URL to the code

Reply:
Leon was supposed to remove poseidon.md and poseidon2.md files from the repo.
Karthik - can you please do that?

Reply:
@jeremyfelder: https://github.com/ingonyama-zk/icicle/tree/danny_poseidon2_docs/v3.2/examples/rust/poseidon2 
it is in the Pr branch though, not yet pushed in main

Reply:
I removed the url